src/rpc/RPCHelpers.cpp

```diff
@@ -1693,4 +1697,4 @@ traverseTransactions(
     return response;
 }
 
-}  // namespace RPC
+}  // namespace RPC
```

Comment:
You may want to mark the checkbox for `Render Final Newline` and `Insert Final Newline` and uncheck `Trim Final Newlines` in VSCode. This way you won't be getting these complaints from github for missing newline at end of file.